packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js

```diff
@@ -19,30 +19,30 @@ exports.setup = function (options, seedLink) {
 const DISEASE_DATA_ELEMENTS = {
   AFR: {
     dataElement: 'PSSS_AFR_Daily_Cases',
-    name: 'Acute fever and rash',
+    name: 'Acute Fever and Rash',
```

Comment:
I'm guessing this migration exists only on the epic branch you're merging in to? In general I am somewhat against editing existing migrations as it can cause issues both on feature/epic branches and in the user's local env, but on the other hand it keeps our migrations themselves cleaner. I think as long as you're conscious about ensure this updated migration get run then I'm okay with it in this instance 👍

Reply:
Thanks for reminding, I guess that is a trade off but I didn't consider that will also effect in the teammates' local dev.